locale/en/about/community.md

```diff
@@ -0,0 +1,53 @@
+It is the mission of CommComm to further build out the Node.js Community. If you're reading this, you're already a part of that community – and as a part of the Node.js Community, we'd love to have your help!
+
+The [nodejs/community-committee](https://github.com/nodejs/community-committee) GitHub repository is a great place to start. Check out the [issues labeled "Good first issue"](https://github.com/nodejs/community-committee/labels/good%20first%20issue) to see where we're looking for help. If you have your own ideas on how we could engage and build the community, feel free to create your own issue.
+
+If you're interested in participating in the Community Committee as a member, you should create an issue asking to be an Observer in our next Community Committee meeting. You can find a great example of such an issue [here](https://github.com/nodejs/community-committee/issues/142).
+
+Before diving in though, please be sure to read the [Collaborator Guide](https://github.com/nodejs/community-committee/blob/master/COLLABORATOR_GUIDE.md).
+
+## Membership
+
+To become a member of the Community Committee, start participating! You must:
+
+ 1. Attend the bi-weekly meetings, investigate issues tagged as good first issue, file issues and pull requests, and provide insight via GitHub.
+ 2. Request to become an Observer by filing an issue. Once added as an Observer to meetings, we will track attendance and participation for 3 months, as according to our governance guidelines.
+ 3. When you meet the minimum attendance and participation expectations, the CommComm will vote to add you as a member.
+
+Membership is for 6 months. The group will ask on a regular basis if the expiring members would like to stay on. A member just needs to reply to renew. There is no fixed size of the CommComm. However, the expected target is between 9 and 12. You can read more about membership, and other administrative details, in our [Governance Guide](https://github.com/nodejs/community-committee/blob/master/GOVERNANCE.md).
+
+Regular CommComm meetings are held bi-monthly in a Zoom video conference, and broadcast live to the public on YouTube. Meeting announcements and agendas are posted before the meeting begins in the organization's [GitHub issues](https://github.com/nodejs/community-committee/issues). Any community member or contributor can ask that something be added to the next meeting's agenda by logging a GitHub Issue.
+
+## Consensus Seeking Process
+
+The CommComm follows a [Consensus Seeking](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making) decision making model.
+
+When an agenda item has appeared to reach a consensus, the moderator will ask "Does anyone object?" as a final call for dissent from the consensus. If a consensus cannot be reached that has no objections then a majority wins vote is called. It is expected that the majority of decisions made by the CommComm are via a consensus seeking process and that voting is only used as a last-resort.
```

Comment:
It may be a bit too early, but [#169 of CommCoom](https://github.com/nodejs/community-committee/issues/169) seems to indicate we all agree on the principle. So maybe we can add a bit on the Condorcet voting?

Reply:
I'd err on the side of landing this PR and then updating once https://github.com/nodejs/community-committee/issues/169 is resolved 👍